dictdiffer/__init__.py

```diff
@@ -388,3 +390,27 @@ def revert(diff_result, destination, in_place=False):
                      and return the destination structure.
     """
     return patch(swap(diff_result), destination, in_place)
+
+
+def assert_no_diff(*args, **kwargs):
```

Comment:
I would see this function in `utils`.

Reply:
adding to `utils` creates a circular import, I added it to a new file testing_utils.py instead.